dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.3.2/bnd.bnd

```diff
@@ -13,13 +13,11 @@
 
 instrument.classesIncludes: org/apache/cxf/*.class
 
-cxfVersion=3.4.1
-
 -includeresource: \
-   @${repo;org.apache.cxf:cxf-rt-bindings-xml;${cxfVersion};EXACT}!/!*-INF/*, \
+   @${repo;org.apache.cxf:cxf-rt-bindings-xml;3.4.1;EXACT}!/!*-INF/*, \
    META-INF=resources/META-INF
 
--buildpath: org.apache.cxf:cxf-rt-bindings-xml;strategy=exact;version=${cxfVersion},\
+-buildpath: org.apache.cxf:cxf-rt-bindings-xml;strategy=exact;version=3.4.1,\
```

Comment:
I changed to hardcoded value while experimenting different versions between bnd.bnd and bnd.overrrides. That's the reason why you see these hardcoded values. Now I can replace them with cxfVersion since they'll stay the same.